includes/functions.php

```diff
@@ -334,6 +344,18 @@ function tsml_custom_post_types()
 	);
 }
 
+//function: unregister custom post types and taxonomies
+//used: 	plugin deactivation, before final rewrite flush
+function tsml_unregister_custom_post_types()
+{
+	unregister_taxonomy('tsml_region');
+	unregister_taxonomy('tsml_location');
+	unregister_taxonomy('tsml_district');
+	unregister_post_type('tsml_meeting');
+	unregister_post_type('tsml_location');
+	unregister_post_type('tsml_group');
+}
```

Comment:
feel like it would be cleaner to merge this with `tsml_plugin_deactivation` instead of creating a new function, since it's only ever called from there

Reply:
i think we should also check `post_type_exists` / `taxonomy_exists` though to prevent errors

Reply:
Sounds good, will move.